tests/testthat/test-unnecessary_lambda_linter.R

```diff
@@ -81,3 +81,34 @@ test_that("purrr-style anonymous functions are also caught", {
     unnecessary_lambda_linter()
   )
 })
+
+test_that("cases with braces are caught", {
+  linter <- unnecessary_lambda_linter()
+  print_msg <- rex::rex("Pass print directly as a symbol to lapply()")
```

Comment:
Nitpick: Everywhere else we are using _lint_msg_ as a variable name

Reply:
used print_ in this case because the message is customized for the print() function in the code, in case there are other tests w different messages